json-templates/vpc-create.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
I just added this file to have an example for VPC policy. Not necessary for this feature.